crates/modelardb_compression/src/compression.rs

```diff
@@ -547,45 +577,45 @@ mod tests {
 
     #[test]
     fn test_try_compress_constant_linear_random_regular_time_series() {
-        let timestamps = test_util::generate_timestamps(3 * TRY_COMPRESS_TEST_LENGTH, false);
-        try_compress_constant_linear_random_time_series(timestamps);
+        let uncompressed_timestamps =
+            data_generation::generate_timestamps(3 * TRY_COMPRESS_TEST_LENGTH, false);
+        try_compress_constant_linear_random_time_series(uncompressed_timestamps);
     }
 
     #[test]
     fn test_try_compress_constant_linear_random_irregular_time_series() {
-        let timestamps = test_util::generate_timestamps(3 * TRY_COMPRESS_TEST_LENGTH, true);
+        let timestamps = data_generation::generate_timestamps(3 * TRY_COMPRESS_TEST_LENGTH, true);
         try_compress_constant_linear_random_time_series(timestamps);
     }
 
-    fn try_compress_constant_linear_random_time_series(timestamps: Vec<i64>) {
-        let mut constant = test_util::generate_values(
-            &timestamps[0..TRY_COMPRESS_TEST_LENGTH],
-            StructureOfValues::Constant,
-            None,
-            None,
+    fn try_compress_constant_linear_random_time_series(uncompressed_timestamps: TimestampArray) {
```

Comment:
Since the main difference between `try_compress_constant_linear_random_time_series` and `try_compress_random_linear_constant_time_series` is the order, maybe create a more generalized function to avoid duplicated code.

Reply:
Fixed.